### PR TITLE
Use short type var names per PEP 8

### PR DESCRIPTION
**Pull request by srittau**
Closes: #1872

### CODE REVIEW

File: third_party/2and3/click/types.pyi [hunk 1]
```diff
@@ -55,7 +55,7 @@ class IntParamType(ParamType):
 class IntRange(IntParamType):
     def __init__(self, min: Optional[int] = ..., max: Optional[int] = ..., clamp: bool = ...) -> None: ...
 
-_PathType = TypeVar("_PathType", str, bytes)
+_PathType = Any
```
Comment (hauntsaninja) [inline third_party/2and3/click/types.pyi:58]:
```suggestion
_PathType = AnyStr
```

**Review by hauntsaninja — COMMENTED** [review]
I personally like the T suffix (https://github.com/python/typeshed/issues/1872#issuecomment-364718407), but whatever :-)

File: third_party/2and3/click/types.pyi (hunk 1 above)
Comment (srittau) [reply third_party/2and3/click/types.pyi:58]: `_PathType` is never used as a type variable, it's just a glorified `Union[str, bytes]` return type. (Which means I can leave my obligatory link to python/typing#566 here.)

**Comment by JelleZijlstra** [comment]
This has a couple of merge conflicts now. I'm not sure I like this change; T is fine for a general-purpose typevar, but I find it clearer to use a more explicit name like `MessageType` for typevars with a bound.

**Comment by hauntsaninja** [comment]
It's also bad user experience in the cases where these types get mentioned in type checker error messages (happened recently where we renamed `_LT` to `_SupportsLessThanT` in https://github.com/python/mypy/issues/9582 / https://github.com/python/typeshed/pull/4579).
I opened #4898 as an alternative for the type variables identified in this PR.

**Pull request closed without merging**